### PR TITLE
Popover with info about missing repos

### LINKED ISSUE #680: Organisations: Link to allow users to review their organisation permissions [3]
The way the GitHub API works for showing organisation repositories is that you ask for the `read:org` scope, and then the OAuth authorisation page asks you which organisations you want the application to be able to see.  If you want to review that list later - and this is quite likely - then you need to go to your [authorised applications](https://github.com/settings/applications) page and configure it there.

The GitHub documentation [recommends](https://developer.github.com/v3/oauth/#directing-users-to-review-their-access-for-an-application) that applications provide a direct link to the GitHub settings page corresponding to their application, so that users can review this sort of thing easily.  We need to work out where to put that.

For comparison, see the [Travis profile page](https://travis-ci.org/profile), which has a bit in the bottom left saying "Is an organization missing?  Review and add your authorized organizations."

### PR DESCRIPTION
**Pull request by bartaz**
Fixes #680

Reusing current popover as discussed [in comments](https://github.com/canonical-ols/build.snapcraft.io/issues/680#issuecomment-297953437) of #680.

screen shot 2017-05-04 at 12 25 24

### CODE REVIEW

File: src/server/helpers/config.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const CLIENT_SIDE_WHITELIST = [
   'NODE_ENV',
   'BASE_URL',
   'GITHUB_API_ENDPOINT',
+  'GITHUB_AUTH_CLIENT_ID',
```
Comment (bartaz) [inline src/server/helpers/config.js]: CLIEND_ID is not secret (it's part of app setting URL), so we can safely expose it to client.

File: src/common/components/private-repos-info/index.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
-import React, { Component } from 'react';
+import React, { Component, PropTypes } from 'react';
```
Comment (earnubs) [inline src/common/components/private-repos-info/index.js]: may be worth starting to use `https://www.npmjs.com/package/prop-types`

**Comment by matthewpaulthomas** [comment]
That is too much stuff for a popover, but I understand the need to do it as a short-term fix.

I specced that the comma-separated list of organizations be in brackets, rather than following a colon, but I guess it doesn’t really matter which.

Apart from that, the screenshot looks fine to me.

File: src/common/components/private-repos-info/index.js (hunk 2 above)
Comment (bartaz) [inline src/common/components/private-repos-info/index.js]: We are not yet on React 15.5 that deprecates that, but that is something to consider sooner then later I guess.

**Comment by bartaz** [comment]
Added #715 as follow up to not loose track of visual update for this feature.

**Pull request merged**